### PR TITLE
feat: Workflow for Customer Approval & Account Approval

### PR DESCRIPTION
**Pull request by anjusha12**
## Is this a Feature, Chore or Bug?
- Feature

## Clearly and concisely describe the feature, chore or bug.
- A new account is disabled by default and needed an approval workflow to be enabled. It will only be enabled once approved.
- A new customer is disabled by default and needed an approval workflow to be enabled. It will only be enabled once approved.

## Solution description 
- Created workflow for Customer Approval and account Approval 

## Output screenshots(optional)
[Screencast from 08-17-2024 10:07:05 AM.webm](https://github.com/user-attachments/assets/511387d9-2c6d-49ef-950c-8724fb9931a6)
[Screencast from 08-17-2024 10:05:24 AM.webm](https://github.com/user-attachments/assets/059124ca-bbfc-4a97-a165-6f80d2bf7b78)

## Areas affected and ensured
-`beams/hooks.py`
-`beams/fixtures/`
-`beams/public/js/account.js`
-`beams/public/js/customer.js`

## Did you test with the following dataset?
- Existing Data
- New Data

### CODE REVIEW

**Review by MhmdSinanKT — CHANGES_REQUESTED** [review]
If both customer and account are disabled by default, then the value of the disable checkbox should be 1 on creation. Use field default in doctype instead of client script. Similarly, use server script to update the value of checkbox.

File: beams/fixtures/role.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "name": "Accounts User",
+  "notifications": 1,
+  "restrict_to_domain": null,
+  "role_name": "Accounts User",
+  "search_bar": 1,
+  "timeline": 1,
+  "two_factor_auth": 0,
+  "view_switcher": 1
+ }
+]
```
Comment (MhmdSinanKT) [inline beams/fixtures/role.json:23]: Why is this in fixture? it is already present in ERPNext

File: beams/hooks.py [hunk 2]
```diff
@@ -30,7 +30,9 @@
 # include js in doctype views
 doctype_js = {
     "Sales Invoice": "public/js/sales_invoice.js",
-    "Quotation": "public/js/quotation.js"
+    "Quotation": "public/js/quotation.js",
+    "Customer": "public/js/customer.js",
```
Comment (MhmdSinanKT) [inline beams/hooks.py:34]: remove these lines if there are no corresponding files

**Review by MhmdSinanKT — APPROVED** [review]

**Review by MhmdSinanKT — CHANGES_REQUESTED** [review]
Change custom fields to work on setup

**Review by SherinKR — APPROVED** [review]

**Pull request merged**